app/models/site_config.rb

```diff
@@ -142,6 +142,8 @@ class SiteConfig < RailsSettings::Base
   field :prefer_manual_suggested_users, type: :boolean, default: false
 
   # Rate limits and spam prevention
+  # NOTE: @citizen428 These will be removed once we migrated to the new settings
```

Comment:
**(nitpick, non-blocking)**

```suggestion
  # NOTE: @citizen428 These will be removed once we've migrated to the new settings
```

Reply:
While you're not wrong, it's phrased like this in other comments to so I guess we'll just live with it for a short time, they'll all be removed soon anyway.